app/models/site.rb

```diff
@@ -20,4 +21,11 @@ class Site < ActiveRecord::Base
   def credit_enabled?
     participants.exists? && credit_roles.exists?
   end
+
+  def validate_category_title_format
+    if /%{(?!name)}/ =~ category_title_format ||
+         /%[^{]/ =~ category_title_format
+      errors.add(:category_title_format, "Invalid format")
```

Comment:
Don't assign message string directly.
Please use symbol here to allow using i18n.